packages/middleware/src/createServer.ts

```diff
@@ -33,6 +37,15 @@ async function createServer<
   config: MiddlewareConfig<TIntegrationContext>,
   options: CreateServerOptions = {}
 ): Promise<Server> {
+  const loggerManager = new LoggerManager<LoggerFactoryConfig>(
+    config,
+    (loggerConfig) => LoggerFactory.create(LoggerType.ConsolaGcp, loggerConfig)
+  );
+  const rawGlobalLogger = loggerManager.getGlobal();
```

Comment:
```suggestion
  const logger = loggerManager.getGlobal();
```
What is the point to distinguish it here with a name? It is the same interface and you assume that you know internals of getter function. All you care about should be to get loggerInterface typed object.